docs/reference/search/search-your-data/learning-to-rank-search-usage.asciidoc

```diff
@@ -0,0 +1,77 @@
+[[learning-to-rank-search-usage]]
+=== Search using Learning To Rank
+++++
+<titleabbrev>Search using Learning To Rank</titleabbrev>
+++++
+
+preview::["The Learning To Rank feature is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but this feature is not subject to the support SLA of official GA features."]
+
+NOTE: This feature is available for Elastic Stack versions 8.12.0 and newer and requires a https://www.elastic.co/pricing[Platinum subscription] or higher.
+
+[discrete]
+[[learning-to-rank-rescorer]]
+==== Learning To Rank as a rescorer
+
+Once your LTR model is trained and deployed into {es}, it is possible to use it as a <<rescore, rescorer>> in the Elasticsearch <<search-your-data, Search API>>:
+
+[source,console]
+----
+GET my-index/_search
+{
+  "query": { <1>
+    "multi_match": {
+      "fields": ["title", "content"],
+      "query": "the quick brown fox"
+    }
+  },
+  "rescore": {
+    "learning_to_rank": {
+      "model_id": "ltr-model", <2>
+      "params": { <3>
+        "query_text": "the quick brown fox"
+      }
+    },
+    "window_size": 100 <4>
+  }
+}
+----
+// TEST[skip:TBD]
```

Comment:
Not sure if it's feasible to test these console snippets because it requires loading an LTR model.